### ISSUE TITLE
Staging Review finding: Results cards receiving keyboard focus

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98566]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the results page, each card can receive keyboard focus when tabbing through the page. With very few exceptions (eg. programmatically assigned focus management), non-interactive elements should not receive focus.

Keyboard users and screen reader users typically expect elements that receive focus to respond to standard keystrokes for the type of element it is, eg. a button responds to [enter]. When a non-interactive element receives focus, users may be confused about the function of the element --- what is it and what actions does it support?

<details><summary>Link, screenshot or steps to recreate</summary>

Example of a card receiving focus:
```
<va-card tabindex="0" class="hydrated">
   <div class="blue-heading">
      <span>
         <b>Time-sensitive benefit</b>
      </span>
   </div>
   <h3 class="vads-u-margin-top--0">
      <span class="usa-label category-label">Disability</span>
      <br><br>
      <span>Disability compensation</span>
   </h3>
   <p class="vads-u-margin-y--0">[content]</p>
   [links]
</va-card>
```

</details>

## Recommended action

The `tabindex="0"` on each card makes it keyboard focusable, and should be removed. 

I think the intention here was to make the cards more discoverable, or possibly to make the context of the learn more/apply now links clearer by forcing screen reader users to tab to the card first and hear the content related to those links. But it's not a good practice to disrupt the normal flow of content in this way.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by nkontrabecki** [comment]
Good Morning. The team will remove the "tabindex=0"  on each card. [Jira ticket 358](https://jira.devops.va.gov/browse/PTEMSVT-358) in Sprint 17.

**Comment by nkontrabecki** [comment]
Addressed in [Jira sprint ticket [](358](https://jira.devops.va.gov/browse/PTEMSVT-358)) in Sprint 17. @briandeconinck, please let us know if this meets the requirement. [Our Staging environment](https://staging.va.gov/discover-your-benefits)

**Comment by nkontrabecki** [comment]
Confirmed fixed and tested in Sprint 17 Jira ticket 358. Also reviewed the implemented fix with @rileyorr Thank you
